docs/glossary.md

```diff
@@ -0,0 +1,100 @@
+# InSpec Glossary
+
+## Basic Syntax
+```
+describe foo('/path/to/foo.txt') do
+    its('blah') { should cmp '123' }
+    it { should exist }
+    it { should be_reasonable }
+    it { should_not be_ridiculous }
+end
+```
+## Basic Elements:
+
+### describe **foo**, where
+
+  * `foo` is the _resource_
+
+### describe foo **('/path/to/foo.txt')**, where
+
+  * `('/path/to/foo.txt')` is the _resource parameter_
+
+## Tests:
+
+### **its('blah') { should cmp '123' }** is an _individual test_, where
+
+  * `'blah'` is a _property_
+  * { should cmp '123' }is a _condition statement_
```

Comment:
Missing a space between `}` and `is`, and `{ should cmp '123' }` should be wrapped in backticks.

Reply:
OK